### PR TITLE
Completed tag

### PR DESCRIPTION
**Pull request by ianmah**
image

### CODE REVIEW

File: components/Evaluator/HackerList.js [hunk 1]
```diff
@@ -76,12 +76,16 @@ export default function HackerList({
...
           <HackerEntry
+            key={applicant._id}
             index={applicant.index || i}
             id={applicant.id || 'tbd'}
             firstName={applicant.basicInfo.firstName}
             lastName={applicant.basicInfo.lastName}
             score={applicant.score}
             selectHacker={() => setSelectedApplicant(applicant)}
+            hasCompleted={
+              applicant.score && Object.keys(applicant.score.scores).length >= 3
```
Comment (andykwan1124) [inline components/Evaluator/HackerList.js:87]: Think this should be just `4`?

**Review by andykwan1124 — COMMENTED** [review]
YO looks great. Thanks for fixing the responsiveness stuff in this PR!!!!! Just a smal nit

**Comment by andykwan1124** [comment]
What did you change to get the responsiveness working lol

**Review by andykwan1124 — APPROVED** [review]

**Comment by ianmah** [comment]
> What did you change to get the responsiveness working lol

Bunch of flexbox tings

**Pull request merged**